.gitignore

```diff
@@ -96,3 +96,6 @@ Pods/
 
 # Artifacts directory
 artifacts/
+
+# Git generated files for conflicting
+*.orig
```

Comment:
No, I want to ignore all files for terminating `orig`, so I think it should be `*.orig`, right?